tests/samplers_tests/test_nsgaii.py

```diff
@@ -865,7 +866,6 @@ def test_crossover_inlined_categorical_distribution() -> None:
         BLXAlphaCrossover(),
         SPXCrossover(),
         SBXCrossover(),
-        VSBXCrossover(),
```

Comment:
Removed because even if the parent has the same score, the child is created in its neighborhood and the purpose of this test, to be the same individual as the parent, is not achieved.